modules/ROOT/pages/configuration/configuration-settings.adoc

```diff
@@ -360,13 +360,59 @@ m|+++10m+++
 |===
 |Description
 a|A comma-separated list of endpoints that a server should contact in order to discover other cluster members.
+Typically, all cluster members, including the current server, must be specified in this list.
+The setting configures the endpoints for Discovery service V2.
+|Valid values
+a|A comma-separated list where each element is a socket address in the format of `hostname:port`, `hostname`, or `:port`.
+|Default value
+m|
+|===
+
+[role=label--enterprise-edition, label--new-5.22]
```

Comment:
```suggestion
[role=label--enterprise-edition label--new-5.22]
```

To specify multiple roles in this format, separate them with a space.

https://docs.asciidoctor.org/asciidoc/latest/attributes/role/#formal-role-syntax-for-blocks